src/exe/cimbar/cimbar.cpp

```diff
@@ -73,51 +173,55 @@ int main(int argc, char** argv)
 	unsigned compressionLevel = cimbar::Config::compression_level();
 	unsigned ecc = cimbar::Config::ecc_bytes();
 	options.add_options()
-	    ("i,in", "Encoded pngs/jpgs/etc (for decode), or file to encode", cxxopts::value<vector<string>>())
-	    ("o,out", "Output file prefix (encoding) or directory (decoding).", cxxopts::value<string>())
-	    ("c,color-bits", "Color bits. [0-3]", cxxopts::value<int>()->default_value(turbo::str::str(colorBits)))
-	    ("e,ecc", "ECC level", cxxopts::value<unsigned>()->default_value(turbo::str::str(ecc)))
-	    ("f,fountain", "Attempt fountain encode/decode", cxxopts::value<bool>())
-	    ("z,compression", "Compression level. 0 == no compression.", cxxopts::value<int>()->default_value(turbo::str::str(compressionLevel)))
-	    ("color-correct", "Toggle decoding color correction. 1 == on. 0 == off.", cxxopts::value<int>()->default_value("1"))
-	    ("encode", "Run the encoder!", cxxopts::value<bool>())
-	    ("no-deskew", "Skip the deskew step -- treat input image as already extracted.", cxxopts::value<bool>())
-	    ("undistort", "Attempt undistort step -- useful if image distortion is significant.", cxxopts::value<bool>())
-	    ("preprocess", "Run sharpen filter on the input image. 1 == on. 0 == off. -1 == guess.", cxxopts::value<int>()->default_value("-1"))
-	    ("h,help", "Print usage")
+		("i,in", "Encoded pngs/jpgs/etc (for decode), or file to encode", cxxopts::value<vector<string>>())
+		("o,out", "Output file prefix (encoding) or directory (decoding).", cxxopts::value<string>())
+		("c,color-bits", "Color bits. [0-3]", cxxopts::value<int>()->default_value(turbo::str::str(colorBits)))
+		("e,ecc", "ECC level", cxxopts::value<unsigned>()->default_value(turbo::str::str(ecc)))
+		("z,compression", "Compression level. 0 == no compression.", cxxopts::value<int>()->default_value(turbo::str::str(compressionLevel)))
+		("color-correct", "Toggle decoding color correction. 1 == on. 0 == off.", cxxopts::value<int>()->default_value("1"))
+		("encode", "Run the encoder!", cxxopts::value<bool>())
+		("no-deskew", "Skip the deskew step -- treat input image as already extracted.", cxxopts::value<bool>())
+		("no-fountain", "Disable fountain encode/decode. Will also disable compression.", cxxopts::value<bool>())
```

Comment:
fountain is now the default. The option to disable it is `--no-fountain`, in line with `--no-deskew` above.